packages/dd-trace/src/proxy.js

```diff
@@ -218,11 +233,11 @@ class Tracer extends NoopProxy {
         this._modules.llmobs.enable(config)
       }
       if (!this._tracingInitialized) {
-        const prioritySampler = appsecStandalone.configure(config)
+        const prioritySampler = config.appsec.standalone?.enabled && require('./appsec/standalone').configure(config)
         this._tracer = new DatadogTracer(config, prioritySampler)
         this.dataStreamsCheckpointer = this._tracer.dataStreamsCheckpointer
-        this.appsec = new AppsecSdk(this._tracer, config)
-        this.llmobs = new LLMObsSDK(this._tracer, this._modules.llmobs, config)
+        lazyProxy(this, 'appsec', () => require('./appsec/sdk'), this._tracer, config)
```

Comment:
this will be lazy loaded even outside of serverless, thus incurring a synchronous cost at runtime. You literally asked to not do this yourself.

You could manually trigger the proxy when not in serverless like that:
```javascript
if (!config._isInServerlessEnvironment()) {
  this.appsec
}
```

Reply:
Yeah, after thinking more about it I figured it's better to be consistent, but I don't wanna cause a regression either. Will add a condition.

Reply:
Done.